packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -110,15 +125,45 @@ class GameWidget<T extends Game> extends StatefulWidget {
     this.focusNode,
     this.autofocus = true,
     MouseCursor? mouseCursor,
-  }) : super(key: key) {
+  })  : create = null,
+        super(key: key) {
     if (mouseCursor != null) {
-      game.mouseCursor = mouseCursor;
+      game!.mouseCursor = mouseCursor;
     }
     if (initialActiveOverlays != null) {
-      game.overlays.addAll(initialActiveOverlays);
+      game!.overlays.addAll(initialActiveOverlays);
     }
   }
 
+  /// Renders a [game] in a flutter widget tree.
```

Comment:
`game` is not an argument in this constructor. So this should say something like "A widget that [create]s a new game instance and then renders it in the Flutter widget tree".